ert_shared/cli/main.py

```diff
@@ -61,17 +61,16 @@ def run_cli(args):
             )
             raise ErtCliError(msg)
 
-        ee_config = EvaluatorServerConfig(custom_port_range=args.port_range)
-        argument.update({"ee_config": ee_config})
+        evaluator_config = EvaluatorServerConfig(custom_port_range=args.port_range)
```

Comment:
I agree that it should be renamed, and that `evaluator_config` is better than `ee_config`, but I wouldn't guess that `evaluator_config` contained only connection information, such as token, cert, protocol etc.. Should we rename it further towards server? I would think `evaluator_config` contained information on how to evaluate the ensemble, and things like `algorithm` would be included in it.

Reply:
`evaluator_server_config` then ? :)